packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/.gitignore

```diff
@@ -0,0 +1,2 @@
+upgrade-walletFactory-permit.json
```

Comment:
I suppose we could. But since bash lacks stuff like "find all references", it's sort of a pain to do. A few hours of trial-and-error. Does it seem worthwhile?

Reply:
not a blocker; something to revisit later. may fall out of other rearrangements.